tests/test_main.py

```diff
@@ -447,3 +447,10 @@ def test_simple_output(filesystem):
     result = CliRunner().invoke(climain, ['run', '--paths-to-mutate=foo.py', "--simple-output"], catch_exceptions=False)
     print(repr(result.output))
     assert '14/14  KILLED 14  TIMEOUT 0  SUSPICIOUS 0  SURVIVED 0  SKIPPED 0' in repr(result.output)
+
+
+def test_simple_output(filesystem):
+    result = CliRunner().invoke(climain, ['run', '--paths-to-mutate=foo.py', "--simple-output"], catch_exceptions=False)
+    print(repr(result.output))
+    assert CliRunner().invoke(climain, ['result-ids', "survived"], catch_exceptions=False).output.strip() == "1"
```

Comment:
I didn't get the tests t run locally, but I tested the changes manually and they work. before merging, one would have to change this line to actually assert the correct behavior (14 IDs separated by spaces).